src/borg/remote.py

```diff
@@ -297,12 +297,12 @@ def negotiate(self, client_data):
         if client_data == RPC_PROTOCOL_VERSION:
             return RPC_PROTOCOL_VERSION
         # clients since 1.1.0b3 use a dict as client_data
+        # clients since 1.1.0b6 support json log format from server
         if isinstance(client_data, dict):
```

Comment:
we do not need the client_supports_log_v3 flag because we can just decide based on the client_data being a dict (which will be the case for 1.1.0b3+) and just stop supporting all 1.1.0 betas.